recipes/nasm/all/conanfile.py

```diff
@@ -11,85 +18,106 @@ class NASMConan(ConanFile):
     homepage = "http://www.nasm.us"
     description = "The Netwide Assembler, NASM, is an 80x86 and x86-64 assembler"
     license = "BSD-2-Clause"
-    settings = "os", "arch", "compiler", "build_type"
     topics = ("nasm", "installer", "assembler")
 
-    exports_sources = "patches/*"
-    _autotools = None
+    settings = "os", "arch", "compiler", "build_type"
+
 
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    def export_sources(self):
+        export_conandata_patches(self)
 
-    @property
-    def _settings_build(self):
-        return getattr(self, "settings_build", self.settings)
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
 
     def configure(self):
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        # for plain C projects only
```

Comment:
Remove comments, not only here, but also from conanfile.py. It's documented to be removed: https://github.com/conan-io/conan-center-index/blob/master/docs/package_templates/autotools_package/all/conanfile.py#L14

Keep FIXME and TODO only.

Reply:
There are a few notes about compiler settings, like the one about GCC9 "pure" attribute.
They were there before I upgraded the recipe, and I think we should leave them there,
but they aren't really TODO or FIXME, just notes about why the line is in there.
Ok to leave those?

Reply:
Yes, it's fine keeping important information for devs, but comments from templates should be removed.